change/@fluentui-web-components-b786537a-e6c4-4798-89b5-b0f60dca78e8.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Revises Divider template to import from FAST Foundation.",
```

The comment becomes a changelog entry
```suggestion
  "comment": "feat(divider): Add divider web component",
```